src/components/countdownTimer/CountdownTimer.tsx

```diff
@@ -12,24 +12,21 @@ interface CountdownTimerProps {
 
 export const CountdownTimer = ({
   date,
-  msgEnded,
+  msgEnded = '',
   withDays = false,
   intervalSeconds = 1,
 }: CountdownTimerProps) => {
   const now = dayjs().unix() * 1000;
   const [countdown, setCountdown] = useState(date ? date - now : -1);
   const timerEnded = countdown <= 0;
+  const interval = intervalSeconds * 1000;
```

Comment:
why not simply accept the interval in ms? I don't even think anyone passes this.

Reply:
Sure